models/migrations/v213.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"xorm.io/xorm"
+)
+
+func addKeypairToPushMirror(x *xorm.Engine) error {
+	type PushMirror struct {
+		PublicKey  string
```

Comment:
`string` has a default length 255, is that enough or is that too long?

Reply:
The current generated public keys have a length of 80 characters.
Private keys on the other hand have a larger length(due to OpenSSH format)... Yeah that needs to be adjusted.